.github/workflows/pull-request-update.yml

```
@@ -72,3 +72,14 @@ jobs:
           COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
           # pass GitHub token to allow accurately detecting a build vs a re-run build
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  run-visual-regression-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+      - run: npm install --save-dev @percy/cli
```

```suggestion
      - run: npm install
```
Percy is already a dev dependency and you probably also need all the other dependencies in order to run the app